### PR TITLE
uploading day2 assignment

### PR DESCRIPTION
**Pull request by Snehapadvi96**

### CODE REVIEW

**Review by Mrunali2997 — APPROVED** [review]

File: Sneha/html/day1.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    <body>
+        <h2>Introduction</h2>
+        <p>
+            I am sneha padvi .I am from nashik <br>
+            I have completed my BCS from North maharashtra university.<br>
+            Now currently I am pursuing MCA from KK Wagh College,Nashik.
+
+            </p>
+    </body>
+</html>
```
Comment (Pawan1005) [inline Sneha/html/day1.html:70]: make 4 separate files

File: Sneha/html/day2.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Day2</title>
+    <style>
+        div{ background-color: peachpuff;}
+
```
Comment (Pawan1005) [inline Sneha/html/day2.html:10]: remove extra lines

File: Sneha/html/day2.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    <p >This article is about the company.</p>
+    <h1>From the garage to the Googleplex</h1>
+     <p> <font size="5">Google story begins in 1995 at <b>Stanford University.</b><br>
+        <Strong>Larry Page </Strong>was considering Stanford for grad school and <b>Sergey Brin,</b> a student
+        there, was assigned to show him around. </font></p>
+        By some accounts, they disagreed about nearly everything during that first meeting, but by the following
+        year they struck a .</br>
+
+
+
```
Comment (Pawan1005) [inline Sneha/html/day2.html:24]: remove extra lines

File: Sneha/html/day2.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        Working from their dorm rooms, they built a search engine that used links to determine the importance of
+        individual pages on the <a href="https://www.goggle.com">World Wide Web </a>.</br>
+        They called this search engine <i>Backrub.</i></p>
+        <hr>
+
+
+
+
+
```
Comment (Pawan1005) [inline Sneha/html/day2.html:33]: remove extra lines

File: Sneha/html/day2.html [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+             At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 
+             <strike>30</strike> <u>20</u>
+             years, until the year 2024.<br>
+             <font style="color:green;"><center>Thank you!!</center></font>
+        </p>
+    </div>
+
+</body>
+
+</html>
```
Comment (Pawan1005) [inline Sneha/html/day2.html:69]: need indentation 
remove extra spaces and lines

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by nikitadamale97 — APPROVED** [review]

**Pull request closed without merging**